README.md

```diff
@@ -356,28 +419,6 @@ Use `temporal.getState()` to access to temporal store!
 
 > While `setState`, `subscribe`, and `destroy` exist on `temporal`, you should not need to use them.
 
-#### **React Hooks**
```

Comment:
This was removed, as it seemed to be a duplicate of `useTemporalStore` example earlier in the docs